torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14709,6 +14742,11 @@ def ref_pairwise_distance(input1, input2):
            sample_inputs_func=sample_inputs_norm_fro,
            dtypes=floating_and_complex_types_and(torch.bfloat16),
            dtypesIfCUDA=floating_and_complex_types_and(torch.float16, torch.bfloat16),
+           supports_forward_ad=True,
+           # torch.autograd.gradcheck.GradcheckError: While computing batched gradients
```

Comment:
Can you open an issue for this? It looks pretty bad.

Reply:
Yep https://github.com/pytorch/pytorch/issues/74677